storage/src/robocorp/storage/_client.py

```diff
@@ -68,9 +68,22 @@ def _handle_error(response: Response):
 
         return _handle_error
 
-    def list_assets(self) -> List[Asset]:
+    def list_assets(
+        self, *, limit: Optional[int], next_page: Optional[str]
```

Comment:
`next_page` in this API seems strange (I get that in the previous API it returned what'd be the next page, but here this is the `current_page` right -- the param should probably be just `page` I guess).

Reply:
Yes, `page` perfectly inlines with `limit`.

Where on the API side with have `next_page` and `page_limit` so we understand what is this about.